src/report.rs

```diff
@@ -44,7 +46,50 @@ impl Report {
       format!("unix:{:?}", platform.os_type).into()
     };
 
-    let backtrace = format!("{:#?}", Backtrace::new());
+    //We skip non-user code frames, including Backtrace::new()
+    const SKIP_FRAMES_NUM: usize = 8;
+    //Code is based on backtrace source
+    const HEX_WIDTH: usize = mem::size_of::<usize>() * 2 + 2;
```

Comment:
I find this calculation confusing. If we have hard coded values for something in the code, it'd be much better to actually explain _why_ they are there, not just reference the authority that generated them